templates/aws/template-variables.rc

```diff
@@ -1,19 +1,19 @@
 # Kubernetes cluster configuration
-export KUBERNETES_VERSION=v1.30.0
-export CONTROL_PLANE_MACHINE_COUNT=                 # e.g. 1
-export WORKER_MACHINE_COUNT=
+export KUBERNETES_VERSION=v1.31.0
+export CONTROL_PLANE_MACHINE_COUNT=1
+export WORKER_MACHINE_COUNT=1
 
 # AWS region
-export AWS_REGION=""                                # e.g. "us-east-2"
+export AWS_REGION="eu-central-1"
 
 # AWS machine configuration
-export AWS_CREATE_BASTION=                          # e.g. "true"
-export AWS_PUBLIC_IP=                               # e.g. "true"
-export AWS_CONTROL_PLANE_INSTANCE_TYPE=             # e.g. "t3.large"
-export AWS_NODE_INSTANCE_TYPE=                      # e.g. "t3.large"
-export AWS_CONTROL_PLANE_ROOT_VOLUME_SIZE=          # in Gigabyte, e.g. 16
-export AWS_NODE_ROOT_VOLUME_SIZE=                   # in Gigabyte, e.g. 16
-export AWS_SSH_KEY_NAME=                            # e.g. "default"
-export AWS_AMI_ID=                                  # e.g. "ami-0ad50e72a79228704"
+export AWS_CREATE_BASTION="true"
+export AWS_PUBLIC_IP="true"
+export AWS_CONTROL_PLANE_INSTANCE_TYPE="t3.large"
+export AWS_NODE_INSTANCE_TYPE="t3.large"
+export AWS_CONTROL_PLANE_ROOT_VOLUME_SIZE=16
+export AWS_NODE_ROOT_VOLUME_SIZE=16
+export AWS_SSH_KEY_NAME="default"
+export AWS_AMI_ID="ami-027b534ab5d0b4886"
```

Comment:
Maybe we can add a description on how this AMI is obtained?

Reply:
Done